drivers/include/nrfx_nfct.h

```diff
@@ -236,6 +236,19 @@ void nrfx_nfct_rx(nrfx_nfct_data_desc_t const * p_rx_data);
 nrfx_err_t nrfx_nfct_tx(nrfx_nfct_data_desc_t const * p_tx_data,
                         nrf_nfct_frame_delay_mode_t   delay_mode);
 
+/**
+ * @brief Function for transmitting an NFC frame with a specified number of bits.
+ *
+ * @param[in] p_tx_data   Pointer to the TX buffer. Unlike in nrfx_nfct_tx, data_len is
```

Comment:
It would be nice to add a doxygen tag when referring to another API function: e.g. `@ref nrfx_nfct_tx`